.travis.yml

```diff
@@ -1,12 +1,10 @@
 language: ruby
 
 rvm:
-  - 2.0.0
-  - 2.1.0
-  - 2.2.0
+  - 2.2
```

Comment:
for this PR: 

`Build #465 (affb050) of jayroh/thredded@master in PR #134 by Gleb Mazovetskiy passed in 5 min 14 sec`

5 mins! Haha Yuuuup